### PR TITLE
first run

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Comment by samsonasik** [comment]
classmap need to be removed:

```diff
-       "classmap": [
-            "vendor/codeigniter4/framework/app/Controllers/BaseController.php"
-        ]
 ```

**Comment by TomasVotruba** [comment]
It seems the `BaseController.php` is never loaded.  When I add it to the `bootstrap.php`, it works as expected.

Which class or methods provides it?

**Comment by samsonasik** [comment]
that's added via Autoloader

https://github.com/codeigniter4/CodeIgniter4/blob/develop/system/Autoloader/Autoloader.php#L96

and registered from Config

https://github.com/codeigniter4/CodeIgniter4/blob/75d4ed47e09834a5db0b48d3877891976f7efe49/app/Config/Autoload.php#L44

**Comment by TomasVotruba** [comment]
Now it runs well :+1: 

I've tried to include it manually and it is enough. Yet that should not be needed.

When I add `class_exists()` the class is loaded propperly. That means the class is not loaded by PHPStan for some reason.
ATM I would not invest more time into this and simply load it via class map. It's already black magic autoloader here, so trying to solve it is not worth it.

The classmap works well :+1: I'd stick with it.

File: bootstrap.php [hunk 1]
```diff
@@ -11,4 +11,7 @@
 
 require 'vendor/codeigniter4/framework/app/Config/Paths.php';
 $paths = new Config\Paths();
-require_once 'vendor/codeigniter4/framework/system/bootstrap.php';
+require_once __DIR__ . '/vendor/codeigniter4/framework/system/bootstrap.php';
+
+// with this check, the class is loaded
+var_dump(class_exists(\App\Controllers\BaseController::class));
```
Comment (samsonasik) [inline bootstrap.php:17]: is there a way to use it in rector bootstrap.php 

```php
spl_autoload_register(function (string $class): void {
    class_exists($class);
 }

```

Comment (TomasVotruba) [reply bootstrap.php:17]: Yes, just:

```bash
class_exists(\App\Controllers\BaseController::class);
```

Comment (samsonasik) [reply bootstrap.php:17]: I am trying to do with dynamic:

```php
spl_autoload_register(function (string $class): void {

    class_exists($class);
   
    // ...
});
```

that's seems overlapped, 

manually seems ok as far as it is loaded 👍 , Thank you

Comment (TomasVotruba) [reply bootstrap.php]: I would not make this complicated, as the class is used in controllers.
Instead, I'd try get codeigniter to use PSR-4 :)

**Pull request merged**